src/Symfony/Component/Console/composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": ">=7.2.5",
+        "ext-pcre": "*",
```

Comment:
Does it really belong to this PR? Console and many other components require PCRE but none require it explicitly

Reply:
I would remove it here. We don't define this requirement in all our components

Reply:
done